### PR TITLE
Handle uncertainty importing from old fda

### PR DESCRIPTION
**Pull request by codymccoy**

### CODE REVIEW

File: ViewModel/StageTransforms/ExteriorInteriorOwnerElement.cs [hunk 1]
```diff
@@ -64,15 +64,15 @@ public void AddNewExteriorInteriorCurve(object arg1, EventArgs arg2)
         {
             List<double> xValues = new List<double>() { 1,2,3,4,5,6 };
             List<double> yValues = new List<double>() { 1,2,3,4,5,6 };
-            UncertainPairedData defaultCurve = DefaultPairedData.CreateDefaultDeterminateUncertainPairedData(xValues,yValues, "Stage", "Flow", "Rating Curve");
+            UncertainPairedData defaultCurve = UncertainPairedDataFactory.CreateDeterminateData(xValues,yValues, "Stage", "Flow", "Rating Curve");
 
             //create save helper
-            Editors.SaveUndoRedoHelper saveHelper = new Editors.SaveUndoRedoHelper(Saving.PersistenceFactory.GetExteriorInteriorManager()
+            Editors.SaveHelper saveHelper = new Editors.SaveHelper(Saving.PersistenceFactory.GetExteriorInteriorManager()
```
Comment (RyanM-RMA) [inline ViewModel/StageTransforms/ExteriorInteriorOwnerElement.cs]: Unnecessary namespacing

File: ViewModel/StageTransforms/RatingCurveElement.cs [hunk 2]
```diff
@@ -62,22 +59,20 @@ public void RemoveElement(object sender, EventArgs e)
         }
 
         public void EditRatingCurve(object arg1, EventArgs arg2)
-        {
-           
-            Editors.SaveUndoRedoHelper saveHelper = new Editors.SaveUndoRedoHelper(Saving.PersistenceFactory.GetRatingManager(),this, (editorVM) => CreateElementFromEditor(editorVM),
+        {       
+            Editors.SaveHelper saveHelper = new Editors.SaveHelper(Saving.PersistenceFactory.GetRatingManager(),this, (editorVM) => CreateElementFromEditor(editorVM),
```
Comment (RyanM-RMA) [inline ViewModel/StageTransforms/RatingCurveElement.cs]: Unnecessary namespacing

**Review by RyanM-RMA — CHANGES_REQUESTED** [review]
A few minor changes requested, mostly namespacing stuff

File: ViewModel/StageTransforms/ExteriorInteriorElement.cs [hunk 3]
```diff
@@ -146,7 +119,7 @@ public override bool Equals(object obj)
             return retval;
         }
 
-        //todo: Refactor: commenting out
+        //todo: do i need this?
```
[the remark this answers is not in the document]
Comment (codymccoy) [reply ViewModel/StageTransforms/ExteriorInteriorElement.cs:122]: I made a ticket to look at this later.

**Pull request merged**